api/src/main/java/io/strimzi/api/kafka/model/listener/arraylistener/GenericKafkaListenerConfiguration.java

```diff
@@ -229,6 +230,17 @@ public void setIpFamilies(List<IpFamily> ipFamilies) {
         this.ipFamilies = ipFamilies;
     }
 
+    @Description("Whether to create the bootstrap service or not. " +
+            "This field can be used with the `loadBalancer` type listener.")
```

Comment:
Maybe it's worth adding that it's created by default (if not specified differently).